### DISCUSSION TITLE
Webstudio AI Prompts Not Working

### DISCUSSION THREAD

**Discussion by inkedmarketing** [webstudio-is/webstudio-community#134] [Q&A]
Hey guys, I just tried to test Webstudio AI Alpha using the default prompts and get the following error:

Unknown Error: Unexpected token ` in JSON at position 0.

When I enter in my own prompt as a test I get this error:

Unknown Error: Invalid prompt data.

**Comment by kof** [comment]
Just tried all 3 example prompts and they worked, I wonder if this is some random network error or consistent. Also did you try on a blank page or had something on it? To further debug it please try it on a blank page

**Comment by inkedmarketing** [comment]
Maybe, I just got it to work. The only thing I noticed that was off was "Body" wasn't selected and when I selected the Body tag it worked. Is that the normal function is that the Body tag in the Navigator must be selected in order for the AI assistant to work?

**Reply by kof** [reply]
Got it reproduced, yes something on canvas must be selected, we must show error if user didn't select anything first.

**Comment by kof** [comment]
Issue created, feel free to follow it there, thanks for the repro

https://github.com/webstudio-is/webstudio/issues/3077
